<!-- BaseController Launch File.  Consists of top-level Base Controller,
     Speed Controller, and Motor Node -->

 <launch>
   <node name="base_controller" 
         pkg="scr_proto" type="base_controller_node">
   </node>

    <node name="arduino_serial"
   		pkg="rosserial_python" type="serial_node.py">
   		<param name="~port" value="/dev/ttyACM1" />
        <param name="baud" value="115200"/>
   </node>
</launch>
